numpy/core/getlimits.py

```diff
@@ -368,6 +374,10 @@ class finfo:
 
     Machine limits for floating point types.
 
+    .. note::
+       finfo.nmant is inaccurate and set to be deprecated in the future.
+       To get the number of bits in the mantissa, use finfo.mant_dig instead.
```

Comment:
Should add double quotes to get code rendering.  You can move this note to the `nmant` attribute documentation.  I don't think it should be on the `finfo` class itself, it is too specific.

Reply:
Done.